content/en/hosting/analytics/tuning-dbt.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Tuning dbt runs for performance"
+weight: 6
+linkTitle: "dbt Tuning"
+description: >
+  Tuning DBT variables for performance
+relatedContent: >
+  core/overview/cht-sync
+  hosting/analytics/environment-variables
+  hosting/analytics/setup-docker-compose
+  hosting/analytics/setup-kubernetes
+---
+
+In production setups with large tables, it can be helpful to control how DBT runs.
+
+## Threads
+
+The `DBT_THREADS` variable can be used to allow dbt to run independent models concurrently in the same process using threads.
+
+This can allow models to update more quickly by reducing the total duration of a dbt run.
+
+More threads means more database processes running concurrently which can create performance issues; this value is per process, so if running separate processes, the total number of concurrent processes in the db will be `DBT_THREADS x the number of processes`.
+For a single process, models can only be run concurrently if they are independent; if model A depends on model B, model A will never be run before model A finishes.
+
+A sensible default is `DBT_THREADS=3`.
+
+## Batching
+
+For large tables, it may take a long time for all rows to be copied from the source table into the base models the first the CHT Sync is run, or if the base models are very out of date. The `DBT_BATCH_SIZE` variable can be used to limit the number of records inserted in a single dbt run, which allows models to catch up to real-time gradually and progressively.
```

Comment:
```suggestion
For large tables, it may take a long time for all rows to be copied from the source table into the base models the first time CHT Sync is run, or if the base models are very out of date. The `DBT_BATCH_SIZE` variable can be used to limit the number of records inserted in a single dbt run, which allows models to catch up to real-time gradually and progressively.
```